internal/chain/version.go

```diff
@@ -0,0 +1,31 @@
+package chain
+
+import (
+	"strings"
+
+	"github.com/Taraxa-project/taraxa-indexer/internal/common"
+	log "github.com/sirupsen/logrus"
+)
+
+const MinimumProtocolVersion = "1.5.0"
+
+func versionStringToInts(version string) (int64, int64, int64) {
```

Comment:
```suggestion
func versionStringToInts(version string) (major, minor, patch int64) {
```
This approach would make it clearer what it returns